os/pm/pm_test.c

```diff
@@ -64,36 +73,50 @@ static int pmtest_kthread(int argc, char *argv[])
 
 	while (elapsed_time.tv_sec <= (start_time.tv_sec + PMTEST_DURATION_IN_SECS)) {
 		sleep(PMTEST_THREAD_SLEEP_TIME);
-		pm_changestate(PMTEST_DOMAIN, s);
+		pm_changestate(PMTEST_DOMAIN, pm_state);
```

Comment:
is pm_changestate triggered with all the available states.?
Bcoz i cannot find the pm_state variable getting updated with various available states for next iteration.